src/main.cpp

```diff
@@ -1,24 +1,58 @@
 #include "trajectory_data_handling/sqlite_querying.hpp"
 #include "trajectory_data_handling/trajectory_file_manager.hpp"
+#include "simp-algorithms/MRPA.hpp"
 
 int main() {
-    trajectory_file_manager::file_manager file_manager{};
-    std::shared_ptr<std::vector<data_structures::Trajectory>> all_trajectories = std::make_shared<std::vector<data_structures::Trajectory>>();
-    sqlite_querying::query_handler::all_trajectories = all_trajectories;
-    trajectory_sql::trajectory_manager trajectory_manager{};
+    trajectory_data_handling::file_manager file_manager{};
+    auto all_trajectories = std::make_shared<std::vector<data_structures::Trajectory>>();
+    trajectory_data_handling::query_handler::all_trajectories = all_trajectories;
+    trajectory_data_handling::trajectory_manager trajectory_manager{};
 
 
 //    trajectory_manager.reset_all_data();
-    trajectory_manager.create_database();
+//    trajectory_manager.create_database();
 //    trajectory_manager.create_rtree_table();
 //    file_manager.load_tdrive_dataset(*alltrajectories);
-    file_manager.load_geolife_dataset(*all_trajectories);
-    trajectory_manager.insert_trajectories_into_trajectory_table(*all_trajectories);
+//    file_manager.load_geolife_dataset(*all_trajectories);
+//    trajectory_manager.insert_trajectories_into_trajectory_table(*all_trajectories);
 //    trajectory_manager.load_database_into_datastructure();
 //    trajectory_manager.print_trajectories(*alltrajectories);
 //    trajectory_manager.load_trajectories_into_rtree();
 //    trajectory_manager.reset_all_data();
 
+
+    data_structures::Trajectory t {};
+    t.locations.emplace_back(data_structures::Location(1, 0, 1, 2));
+    t.locations.emplace_back(data_structures::Location(2, 1, 4, 2));
+    t.locations.emplace_back(data_structures::Location(3, 4, 7, 4));
+    t.locations.emplace_back(data_structures::Location(4, 12, 17, 9));
+    t.locations.emplace_back(data_structures::Location(5,15, 26, 12));
+    t.locations.emplace_back(data_structures::Location(6, 18, 32, 5));
+    t.locations.emplace_back(data_structures::Location(7,22, 69, 3));
+    t.locations.emplace_back(data_structures::Location(8, 23, 110, 21));
+    t.locations.emplace_back(data_structures::Location(9, 24, 112, 23));
+    t.locations.emplace_back(data_structures::Location(10, 28, 143, 28));
+    t.locations.emplace_back(data_structures::Location(11, 30, 145, 31));
+    t.locations.emplace_back(data_structures::Location(12, 35, 156, 56));
+    t.locations.emplace_back(data_structures::Location(13, 37, 169, 61));
+    t.locations.emplace_back(data_structures::Location(14,41, 173, 66));
+    t.locations.emplace_back(data_structures::Location(15, 44, 182, 69));
+    t.locations.emplace_back(data_structures::Location(16,50, 243, 98));
+    t.locations.emplace_back(data_structures::Location(17, 53, 277, 143));
+    t.locations.emplace_back(data_structures::Location(18, 60, 300, 200));
```

Comment:
skulle vi ikke bruge  data fra datasets?

Reply:
Jo, men vi havde ikke noget data. Vi lavede et simpelt trajectory, hvor vi kendte alle værdierne og kunne skrue på punkterne for at teste om det hele virkede. Vi er ikke nået det punkt endnu, hvor vi begynder at fodre den med actual data.